### ISSUE TITLE
GSSAPI(Kerberos) authentication

### ISSUE THREAD

**Issue by nullcaster** [devshawn/kafka-gitops#18]
Hi,
We are using self-hosted kafka cluster with GSSAPI(Kerberos) authentication.
Can I add this feature and make a PR?

**Comment by devshawn** [comment]
Yes, that would be great! I would happily accept a PR for this contribution 😄

**Comment by devshawn** [comment]
@nullcaster - Did you ever end up working on this?

**Comment by nullcaster** [comment]
@devshawn  - Sorry, I missed it. I will try to do it in the near future.
